packages/botfuel-dialog/src/adapters/adapter.js

```diff
@@ -81,55 +77,45 @@ class Adapter {
    * Adds the user if necessary.
    * Calls the corresponding method of the brain.
    * Adapters can add specific behaviour.
-   * @async
-   * @param {int} userId - the user id
-   * @returns {Promise.<void>}
    */
-  async addUserIfNecessary(userId) {
+  async addUserIfNecessary(userId: string): Promise<void> {
     logger.debug('addUserIfNecessary', userId);
     await this.bot.brain.addUserIfNecessary(userId);
   }
 
   /**
    * Sends a single bot message to the messaging platform.
    * @abstract
-   * @async
-   * @param {Object} botMessage - the bot message
-   * @returns {Promise.<void>}
    */
-  async sendMessage(botMessage) {
+  async sendMessage(botMessage: BotMessageJson): Promise<void> {
     logger.debug('sendMessage', botMessage);
     throw new MissingImplementationError();
   }
 
   /**
-   * Extends a message with extra properties
+   * Extend a message with extra properties
```

Comment:
Should be Extends, see Javadoc best practices ... unless JSDoc has specific ones.

Reply:
Oh you're right, my bad...